db/migrate/20170621180611_add_population_to_budget_headings.rb

```diff
@@ -0,0 +1,5 @@
+class AddPopulationToBudgetHeadings < ActiveRecord::Migration
+  def change
+    add_column :budget_headings, :population, :integer, default: 0
```

Comment:
I think a default to nil would be better and less problem prone, so only if present population stats are calculated. WDYT @voodoorai2000